packages/workflow/src/records/handler/eventNotificationHandler.ts

```diff
@@ -0,0 +1,150 @@
+    `/Practitioner/${system.practitionerId}`
+  )
+
+  const { name, username, type } = system
+  const systemInformationJSON = { name, username, type }
+
+  const taskWithRegLastUser = addExtensionsToTask(unsavedTask, [
+    {
+      url: 'http://opencrvs.org/specs/extension/regLastUser',
+      valueReference: {
+        reference: `Practitioner/${practitioner.id}`
+      }
+    }
+  ])
+
+  const event = getEventType(bundle)
+  const trackingId = await generateTrackingIdForEvents(event, bundle, token)
+
+  const taskWithRegLastUserAndStatus: Task = {
+    ...taskWithRegLastUser,
+    lastModified: new Date().toISOString(),
+    businessStatus: {
+      coding: [
+        {
+          system: 'http://opencrvs.org/specs/reg-status',
+          code: 'IN_PROGRESS'
+        }
+      ]
+    },
+    identifier: [
+      ...taskWithRegLastUser.identifier,
+      {
+        system: `http://opencrvs.org/specs/id/${
+          event.toLowerCase() as Lowercase<EVENT_TYPE>
+        }-tracking-id`,
+        value: trackingId
+      },
+      {
+        system: 'http://opencrvs.org/specs/id/system_identifier',
+        value: JSON.stringify(systemInformationJSON)
+      }
+    ]
+  }
+
+  const officeExtension = taskWithRegLastUserAndStatus.extension.find(
+    (e) => e.url === 'http://opencrvs.org/specs/extension/regLastOffice'
+  ) as
+    | StringExtensionType['http://opencrvs.org/specs/extension/regLastOffice']
+    | undefined
+
+  const officeId = officeExtension?.valueReference.reference.split('/')[1]
+
+  if (!officeId) throw internal('Office id not found in bundle')
+
+  const officeLocationExtension = taskWithRegLastUserAndStatus.extension.find(
+    (e) => e.url === 'http://opencrvs.org/specs/extension/regLastLocation'
+  ) as
+    | KnownExtensionType['http://opencrvs.org/specs/extension/regLastLocation']
+    | undefined
+
+  const officeLocationId =
+    officeLocationExtension?.valueReference.reference.split('/')[1]
+
+  if (!officeLocationId)
+    throw internal('Office location id not found in bundle')
+
+  const office = await getLocationOrOfficeById(officeId)
+  const officeLocation = await getLocationOrOfficeById(officeLocationId)
+
+  const savedBundleWithRegLastUserAndBusinessStatus = {
+    ...bundle,
+    entry: [
+      ...bundle.entry.filter((e) => e.resource.resourceType !== 'Task'),
+      {
+        fullUrl: bundle.entry.find((e) => e.resource.resourceType === 'Task')
+          ?.fullUrl,
+        resource: taskWithRegLastUserAndStatus
+      },
+      ...[office, officeLocation, practitioner].map((r) =>
+        resourceToSavedBundleEntry(r)
+      )
+    ]
+  }
+
+  const responseBundle = await sendBundleToHearth(
+    savedBundleWithRegLastUserAndBusinessStatus
+  )
+  const savedBundle = toSavedBundle(
+    savedBundleWithRegLastUserAndBusinessStatus,
+    responseBundle
+  ) as ValidRecord
+
+  await indexBundle(savedBundle, token)
+  await auditEvent('sent-notification', savedBundle, token)
```

Comment:
We were probably handling the "sent-notification" event in "create-record" endpoint as well. We can remove that now